x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleExplainResponseTests.java

```diff
@@ -162,11 +168,15 @@ protected IndexLifecycleExplainResponse mutateInstance(IndexLifecycleExplainResp
                 break;
             case 10:
                 return IndexLifecycleExplainResponse.newUnmanagedIndexResponse(index);
+            case 11:
+                isTransitiveError = true;
+                failedStepRetryCount = randomInt(13);
```

Comment:
This could randomly cause a test failure if the count were the same, it should use

```suggestion
                failedStepRetryCount = randomValueOtherThan(failedStepRetryCount, () -> randomInt(13));
```